src/marshmallow_sqlalchemy/convert.py

```diff
@@ -147,13 +165,15 @@ def property2field(self, prop, *, instance=True, field_class=None, **kwargs):
         if not instance:
             return field_class
         field_kwargs = self._get_field_kwargs_for_property(prop)
-        field_kwargs.update(kwargs)
+        _field_update_kwargs(field_class, field_kwargs, kwargs)
         ret = field_class(**field_kwargs)
         if (
             hasattr(prop, "direction")
             and self.DIRECTION_MAPPING[prop.direction.name]
             and prop.uselist is True
         ):
+            related_kwargs = self.get_base_kwargs()
+            _field_update_kwargs(RelatedList, related_kwargs, kwargs)
```

Comment:
I wasn't totally sure how I should handle this, but doing it this way will put the `column` attribute into the `metadata` when `ret` is an instance of `Related` (as @AbdealiJK was experiencing in #361).